app/jobs/modsulator_job.rb

```diff
@@ -121,6 +115,40 @@ def update_metadata(druid, xml_string, log)
     end
   end
 
+  # Open a new version for the given object if it is in the accessioned state.
+  # @param   [Dor::Item]  dor_object          The object to version
+  # @param   [String]     original_filename   The name of the uploaded file
+  # @param   [String]     user_login          The current user_login
+  # @param   [File]       log                 Log file handle
+  def version_object(dor_object, original_filename, user_login, log)
+    if(accessioned(dor_object))
```

Comment:
i would do space or parens, but not both.  i.e., i'd vote for leaving it as-is, or switching to `if accessioned(dor_object)`.  i guess the latter would be preferable just for consistency's sake.